spec/index.md

```diff
@@ -2595,7 +2641,7 @@ These are the synchronous read messages:
         sender : UserId;
         paths : List(Path);
       }
-      | CanisterQuery = {
+      | CanisterSignedQuery = {
```

Comment:
I think this should be `CanisterQuery` now that signatures are just returned as part of the normal query, right? (similarly below)

Reply:
Done.